src/components/Cart/Steps/Steps.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="np-cart__steps__container">
+    <div v-for="(element,index) in steps" :key="index" class="np-cart__step">
+      <div v-if="stepValue - 1 === index" class="np-cart__step__circle--active"/>
+      <div v-if="stepValue - 1 > index" class="np-cart__step__circle--completed"/>
+      <div v-if="stepValue - 1 < index" class="np-cart__step__circle--waiting"/>
+      <p v-if="stepValue-1 > index" class="text-primary">{{ element }}</p>
```

Comment:
jezeli podpieliscie `i18n`, to w miejsce `element` powinna pojawic sie translacja